prac_07/project.py

```diff
@@ -0,0 +1,38 @@
+"""
+prac_07 - project.py
+
+Estimated time: 60 mins
+Actual time: 10 mins
+"""
+
+
+class Project:
+    def __init__(self, name, start_date, priority, cost_estimate, completion_percentage):
+        """Initialise Project instance."""
+        self.name = name
+        self.start_date = start_date
+        self.priority = priority
+        self.cost_estimate = cost_estimate
+        self.completion_percentage = completion_percentage
+
+    def __repr__(self):
+        """Return string representation of data for Project."""
+        return (
+            f"{self.name}, start: {self.start_date}, priority {self.priority}, estimate: ${self.cost_estimate:,.2f}"
```

Comment:
While it may violate PEP8, would it be more appropriate to have this string on on line?

Reply:
Can you tell me your reasoning on why we should violate PEP 8? @alexchristie12

Reply:
My only real reasoning was that the string seems to read a bit better on one line, that's all.